app/helpers/application_helper.rb

```diff
@@ -179,8 +191,18 @@ def holding_location(holding)
   # @return [String] the markup
   def holding_block_search(document)
     block = ''
-    links = search_links(document['electronic_access_1display'])
-    holdings_hash = JSON.parse(document['holdings_1display'] || '{}')
+    portfolio_links = electronic_portfolio_links(document)
+    # Create stub holdings for Alma's electronic profiles so the below logic will work.
+    # Alma does not have an 'Online' library for holdings, so we fake it to keep
+    # the old logic around.
+    # @TODO: Remove when Voyager is gone.
+    portfolio_holdings = Hash[
+      portfolio_links.map do |_portfolio|
+        [SecureRandom.uuid, { 'library' => 'Online' }]
```

Comment:
This looks like it's used as the holding id. A big meaningless random number seems like it could be confusing some day. I could imagine debugging something in web tools and trying to track down the target of this id. Maybe we should add the portfolio holding id here https://github.com/pulibrary/bibdata/blob/91666d6c17eb1cb0823d57d5e32c3e986ab92816/marc_to_solr/lib/electronic_portfolio_builder.rb#L24

Reply:
I'm happy to do that, but it would mean this code won't work until we perform a full reindex, and I think we'll likely remove the requirement of having a holding ID in the future. Is that still what you'd like me to do?